### PR TITLE
Finished project

### PR DESCRIPTION
**Pull request by nejidevelops**
On this PR, I have implemented:
- The homepage
- The about page
- Responsive layout
- DOM and Javascript events

Here are screenshots of how my project looks like

Screenshot from 2023-06-28 18-22-10

Screenshot from 2023-06-28 18-22-14

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+          </div>
+        </div>
+      </nav>
+    </header>
+
+    <section class="home">
+      <div class="home-text">
+        <h2>"The IT World Needs You"</h2>
+        <h1>We Build, We Develop</h1>
+        <p class="home-details">Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatem enim obcaecati praesentium veniam nisi suscipit error, modi quisquam eaque fugit asperiores at itaque amet blanditiis mollitia pariatur?</p>
```
Comment (codecaiine) [inline index.html:63]: - [ ] Good job so far, but I think you can adjust the width of this paragraph and make it similar to the original design in order to make it designed to align with the original one 👍 

##### The original design
headig

##### Your design
heading

File: index.html [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+  <main>
+    <header>
+      <nav>
+        <div class="nav-socials">
+          <img src="./images/facebook.png" alt="">
+          <img src="./images/twitter.png" alt="">
+          <a href="#">English</a>
+          <a href="#">My page</a>
+          <a href="#">Logout</a>
+        </div>
```
Comment (codecaiine) [inline index.html:20]: - [ ] I think you can make this top bar smaller like on the original design, please reduce the height a bit 👍 

##### Your design
navv

#### The original design
navv

File: index.html [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+          </div>
+        </div>
+      </div>
+      <a href="#">Join Our Program</a>
+    </section>
+
+    <section class="speakers">
+      <h2>Featured Speakers</h2>
+      <div id="speaker-cards"></div>
+    </section>
```
Comment (codecaiine) [inline index.html]: - [ ] Good job so far your contributor section is loading dynamically :clap:  But the design is a bit different compared to the original design. :call_me_hand:   Please make sure that your contributor section has use the same font, `height`, `width `, and spacing `padding/margin `. And  your `background color`  is white, please change it  to the color defined on the original design  in order to make your  design similar to the original design :+1: 
 Check the screenshots below :point_down:  to see the differences.

#### Orignal design
Screenshot from 2022-10-28 18-19-46

#### Your design
speakers

File: index.html [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+          <div class="logo-title">
+            <h4>Vue.js</h4>
+          </div>
+          <div class="logo-description">
+            <p>Vue.js is a JavaScript framework for building user interfaces and single-page applications.</p>
+          </div>
+        </div>
+      </div>
+      <a href="#">Join Our Program</a>
+    </section>
```
Comment (codecaiine) [inline index.html]: - [ ] Good job so far :clap: But I think there are a few things with your design you need to work on. Please make sure that when the topic card has hovered with the mouse you have a border around it like on the screenshot below and also your button  `Join our program` has the same color as on the design. :+1: 

- [ ] Please make sure to have the same design, especially the space between your program card as defined on the original design :+1: 

#### Orignal design
Template2

#### Your design
progam


- [ ] Please make sure that your program section mobile version design and button are similar to the original. You can see a lot of differences in the screenshots below :point_down: 

#### Original design
mobk

#### Your design
mob

File: index.html [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+    <section class="partners">
+      <h2>Partners</h2>
+      <div class="partner-logos">
+        <img src="./images/apple-13.svg" alt="">
+        <img src="./images/google-1-1.svg" alt="">
+        <img src="./images/meta-1.svg" alt="">
+        <img src="./images/microsoft.svg" alt="">
+        <img src="./images/oracle-6.svg" alt="">
+      </div>
+    </section>
```
Comment (codecaiine) [inline index.html]: - [ ] Please make sure that your sponsor section has the same design( Height, spacing) as the original design. Also, make sure that your images have the same opacity and size as on the original design in order to make your website design wonderful 👍 

#### Your design
sponsors

#### The original design
sponsors

File: index.html (hunk 3 above)
Comment (codecaiine) [inline index.html:139]: - [ ] I would like to recommend you work on all your section and page mobile design. For example the contributor section, there are a few differences between your design and the original one. Please fix the design for all your mobile version for all sections and page:+1: 
#### Original design
Screenshot from 2022-10-28 19-20-02

File: index.html [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+          <i class="fa-brands fa-instagram socials"></i>
+        </a>
+        <a href="">
+          <i class="fa-brands fa-twitter socials"></i>
+        </a>
+        <a href="">
+          <i class="fa-brands fa-youtube socials"></i>
+        </a>
+      </div>
+    </footer>
```
Comment (codecaiine) [inline index.html]: - [ ] Please make sure that your footer design is similar to the original design for both the home and about pages. There are a few differences. Please note that it is very important to stick to the original design as much as possible :smile: 

#### Your design
footer


#### The original design
Screenshot from 2022-10-28 18-41-53

File: about.html [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+      <div class="about-home-text">
+        <h2>"Be among the Best Developers"</h2>
+        <h1>Get Good at Building Websites</h1>
+        <p class="about-home-details">Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatem enim obcaecati praesentium veniam nisi suscipit error, modi quisquam eaque fugit asperiores at itaque amet blanditiis mollitia pariatur?</p>
+        <div class="date-time">
+          <h3>Please contact us per Email for any further questions about DEVSPARK</h3>
+          <p><a href="mailto:newtonombese1@gmail.com">info@devspark.com</a></p>
+        </div>
+      </div>
+    </section>
```
Comment (codecaiine) [inline about.html]: - [ ] Please make sure your about page headline section elements have the same `font size`, `color`, `height`, `width`, and spacing `padding/spacing` as defined in the original design. For example, your headline title needs to be `uppercase`. Please note that is very important to stick to the original design as much as possible :+1: 

Check the screenshots below to compare and see the differences :point_down: 

####  Original design
Screenshot from 2022-10-28 18-44-06

#### Your design
about

File: about.html [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+          </div>
+        </div>
+        <div class="image-2">
+          <div class="image-text-2">
+            <h2>2023</h2>
+            <p>Devspark Summit @ iHub</p>
+          </div>
+        </div>
+      </div>
+    </section>
```
Comment (codecaiine) [inline about.html]: - [x] Good job but your about page design is a bit different from the original design :cry: You can check the difference with the screenshots I'm sharing with you below :arrow_down:  Please make sure to fix it in order to make your design work as expected :+1: 

#### Your design
about


#### Original design
about_design

File: README.md [hunk 9]
```diff
@@ -0,0 +1,143 @@
...
+
+### Key Features <a name="key-features"></a>
+- **Understanding Git, GitHub and GitHub Workflow**
+- **Configuring Linters**
+- **Changing design template to code**
+- **Creating mobile responsive portfolio website**
+- **Interacting with the DOM**
+
+## 🚀 Live Demo <a name="live-demo"></a>
+[Devspark Website](https://newtonombese.github.io/Newton-Capstone-Project/)
```
Comment (codecaiine) [inline README.md:55]: - [ ] Your project demo is not working well great job 😢  Please fix it and make all the pages work as expected in order to make your demo work as expected and your readme more professional 👍

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @NewtonOmbese , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Good project implementation :heavy_check_mark: 
- No linters errors :heavy_check_mark: 
- Good use of github flow :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (nejidevelops) [reply index.html:63]: Fixed :+1:

File: index.html (hunk 2 above)
Comment (nejidevelops) [reply index.html:20]: Fixed :100:

File: index.html (hunk 3 above)
Comment (nejidevelops) [reply index.html]: Fixed

File: index.html (hunk 4 above)
Comment (nejidevelops) [reply index.html]: Fixed :100:

File: index.html (hunk 5 above)
Comment (nejidevelops) [reply index.html]: Fixed :+1:

File: index.html (hunk 6 above)
Comment (nejidevelops) [reply index.html]: Fixed

File: about.html (hunk 7 above)
Comment (nejidevelops) [reply about.html]: Fixed

File: about.html (hunk 8 above)
Comment (nejidevelops) [reply about.html]: Fixed

File: index.html (hunk 3 above)
Comment (nejidevelops) [reply index.html:139]: Fixed

File: README.md (hunk 9 above)
Comment (nejidevelops) [reply README.md:55]: Fixed

**Review by Ofarouq310 — APPROVED** [review]
Hi @NewtonOmbese, ✋ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🥇 

giphy

## Highlights
✅ Milestone works as expected.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**